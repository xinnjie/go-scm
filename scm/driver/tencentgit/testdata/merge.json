{
    "labels": [],
    "id": 339869,
    "title": "edit README",
    "target_project_id": 179129,
    "target_branch": "master",
    "source_project_id": 179129,
    "source_branch": "mr-test",
    "state": "merged",
    "merge_status": "can_be_merged",
    "iid": 1,
    "description": "edit README",
    "created_at": "2021-08-22T16:22:38+0000",
    "updated_at": "2021-08-22T16:29:28+0000",
    "resolved_at": "2021-08-22T16:29:28+0000",
    "merge_type": "SquashAndMerge",
    "assignee": {
        "id": 176405,
        "username": "xinnjie",
        "web_url": "https://git.code.tencent.com/u/xinnjie",
        "name": "xinnjie",
        "state": "active",
        "avatar_url": "https://git.code.tencent.com/uploads/user/avatar/176405/7e72ab97e19e430287c34c54af3bf7e1."
    },
    "author": {
        "id": 176405,
        "username": "xinnjie",
        "web_url": "https://git.code.tencent.com/u/xinnjie",
        "name": "xinnjie",
        "state": "active",
        "avatar_url": "https://git.code.tencent.com/uploads/user/avatar/176405/7e72ab97e19e430287c34c54af3bf7e1."
    },
    "merge_commit_sha": "a398282a168e6919e8ff3806e1fa11f66ab53d30",
    "milestone": null,
    "necessary_reviewers": null,
    "suggestion_reviewers": null,
    "base_commit": "652a91b83cf37786c64eb286f1efba453a15f34f",
    "target_commit": "652a91b83cf37786c64eb286f1efba453a15f34f",
    "source_commit": "26a65da1a8979640c67b099a4ed2538b9feddee2",
    "project_id": 179129,
    "work_in_progress": false,
    "upvotes": 0,
    "downvotes": 0
}