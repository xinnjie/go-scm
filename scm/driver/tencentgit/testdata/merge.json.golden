{
    "Number": 1,
    "Title": "edit README",
    "State": "opened",
    "Body": "edit README\"",
    "Sha": "26a65da1a8979640c67b099a4ed2538b9feddee2",
    "Ref": "refs/merge-requests/1/head",
    "Source": "fix",
    "Target": "master",
    "Link": "https://git.code.tencent.com/xinnjie/testme/merge_requests/1",
    "Closed": true,
    "Merged": false,
    "Mergeable": true,
    "MergeableState": "mergeable",
    "Draft": true,
    "Author": {
        "ID": 13356,
        "Login": "dblessing",
        "Name": "Drew Blessing",
        "Email": "",
        "Avatar": "https://secure.gravatar.com/avatar/b5bf44866b4eeafa2d8114bfe15da02f?s=80\u0026d=identicon"
    },
    "Assignees": [{
        "ID": 12,
        "Login": "axel.block",
        "Name": "Miss Monserrate Beier",
        "Avatar": "http://www.gravatar.com/avatar/46f6f7dc858ada7be1853f7fb96e81da?s=80&d=identicon"
    }],
    "Created": "2015-12-18T18:29:53.563Z",
    "Updated": "2015-12-18T18:30:22.522Z",
    "Fork": "diaspora/diaspora",
    "Head": {
        "Ref": "fix",
        "Sha": "12d65c8dd2b2676fa3ac47d955accc085a37a9c1",
        "Repo": {
            "ID": "32732",
            "Namespace": "diaspora",
            "Name": "diaspora",
            "FullName": "diaspora/diaspora",
            "Perm": {
                "Pull": true,
                "Push": false,
                "Admin": false
            },
            "Branch": "master",
            "Private": false,
            "Clone": "https://gitlab.com/diaspora/diaspora.git",
            "CloneSSH": "git@gitlab.com:diaspora/diaspora.git",
            "Link": "https://gitlab.com/diaspora/diaspora",
            "Created": "0001-01-01T00:00:00Z",
            "Updated": "0001-01-01T00:00:00Z"
        }
    },
    "Base": {
        "Ref": "master",
        "Repo": {
            "ID": "32732",
            "Namespace": "diaspora",
            "Name": "diaspora",
            "FullName": "diaspora/diaspora",
            "Perm": {
                "Pull": true,
                "Push": false,
                "Admin": false
            },
            "Branch": "master",
            "Private": false,
            "Clone": "https://gitlab.com/diaspora/diaspora.git",
            "CloneSSH": "git@gitlab.com:diaspora/diaspora.git",
            "Link": "https://gitlab.com/diaspora/diaspora",
            "Created": "0001-01-01T00:00:00Z",
            "Updated": "0001-01-01T00:00:00Z"
        }
    }
}
